aomi/seed.py

```diff
@@ -483,14 +526,35 @@ def approle(client, approle_obj, opt):
     client.create_role(name, **role_obj)
 
 
+def generated_key(key, opt):
```

Comment:
Maybe have this take a map of generators. Or have it introspect and search for any classes that are loaded that implement `gen_type` and generate.
Then you could have a n8ce littpe decorator to extend it with like
```
Import aomi.generators

@generator("super_random")
 Def random_gen (random_com_api_key: ''):
    Shutil.exec ("curl -H 'api_key: " + random_com_api_key + "' http://random.com")